llms/mlx_lm/generate.py

```diff
@@ -145,16 +150,19 @@ def main():
 
     formatter = colorprint_by_t0 if args.colorize else None
 
-    generate(
-        model,
-        tokenizer,
-        prompt,
-        args.max_tokens,
-        verbose=True,
-        formatter=formatter,
-        temp=args.temp,
-        top_p=args.top_p,
-    )
+    if generate_embeddings:
+        generate_embeddings(model, tokenizer, prompt)
```

Comment:
Seems pretty unlikely to me that someone would use this in the CLI, so maybe we don't add it here? Wdyt?

Reply:
Yea that does seem unlikely, I can update the server file, that makes the most sense.